js/expose.js

```diff
@@ -0,0 +1,35 @@
+function deliverError(name, seq, message, stack) {
+	const error = new Error(message);
+	error.stack = stack;
+	window[name].callbacks.get(seq).reject(error);
+	window[name].callbacks.delete(seq);
+}
+
+function deliverResult(name, seq, result) {
+	window[name].callbacks.get(seq).resolve(result);
+	window[name].callbacks.delete(seq);
+}
```

Comment:
Maybe these two functions have the same risk of name conflict. `chromedp.CallFunctionOn` accepts function declaration. So you can avoid defining them with `page.AddScriptToEvaluateOnNewDocument`.